shell/client/admin-client.js

```diff
@@ -1182,11 +1182,24 @@ const adminRoute = RouteController.extend({
                   let looksGood;
                   if (error) {
                     looksGood = false;
+                    console.log("Sandstorm WILDCARD_HOST self-test failed. Details:");
+                    console.log(error);
+                    console.log(
+                      "Further details may be available in the JS console provided by your web " +
```

Comment:
This message is somewhat confusing to me. The only place this message ever shows up is in the JS console, right? So why is it telling me to check the JS console? I can imagine someone reading this and thinking: "Is there another super secret JS console that I don't know about that has all the answers?!"